### PR TITLE
update w9 endpoint

### PR DESCRIPTION
**Pull request by whatSocks**
Up here for discussion

these are a lot of changes but it's mostly my auto-prettifier. 

Actual changes:
 function serializeAmbassador(ambassador) 

  .put('/ambassadors/:ambassadorId/has_w9/:has_w9', (req, res) => {
    if (!req.authenticated) return _401(res, 'Permission denied.')
    if (!req.admin) return _403(res, 'Permission denied.')
    return updateW9Ambassdor(req, res)
  })

### CODE REVIEW

File: server/app/models/va/SocialMatch.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
+/*
+ *
+ * This model corresponds to the SocialMatch neo4j nodes
```
Comment (brodavi) [inline server/app/models/va/SocialMatch.js]: Just a note for the future: everything in this /models/va/ directory is only useful in the context of the neode ogm. As much as possible, we are trying to move to a more raw cypher database interaction, reducing our use of neode.

File: server/app/routes/api/v1/va/ambassadors.js [hunk 2]
```diff
@@ -225,24 +246,46 @@ async function disapproveAmbassador(req, res) {
...
+  }, 100)
+
+  return _204(res)
+}
+
+/*
+ *
+ * updateAmbassdaorW9status(req, res)
+ *
+ * This function makes an admin out of an Ambassador. Ambassadors are listed in the admin panel,
```
Comment (brodavi) [inline server/app/routes/api/v1/va/ambassadors.js]: I think something got a little mixed up here maybe?

I see `updateW9Ambassdaor` defined above, and it looks almost identical?

**Review by brodavi — CHANGES_REQUESTED** [review]
possible duplication? updateW9Ambassdor and updateAmbassdorW9status which looks like it might be unused?

File: server/app/routes/api/v1/va/ambassadors.js (hunk 2 above)
Comment (whatSocks) [reply server/app/routes/api/v1/va/ambassadors.js]: removed

File: server/app/models/va/SocialMatch.js (hunk 1 above)
Comment (whatSocks) [reply server/app/models/va/SocialMatch.js]: for specifying the models it seems a lot safer to keep neode, but minimize its use aside from that.

**Comment by whatSocks** [comment]
test with frontend here https://github.com/colab-coop/HelloVoter-admin-ui/pull/15

**Review by brodavi — APPROVED** [review]
awesome!

**Pull request merged**